.dev/tests/jest/setup-globals.js

```diff
@@ -25,3 +25,244 @@ global.coblocksLayoutSelector = {
 	categories: [],
 	layouts: [],
 };
+
+// Mock site design state for tests.
+// Disable reason: sorting keys for this object is superfluous.
+/* eslint-disable sort-keys */
+global.siteDesign = {
+	availableDesignStyles: {
+		traditional: {
+			slug: 'traditional',
+			label: 'Traditional',
+			url: 'http://localhost:8888/wp-content/themes/go/dist/css/design-styles/style-traditional.css',
```

Comment:
`wp-env` uses `localhost:8889` for tests, right?

Reply:
Ooo. Hmm, good question.